project-novica/demo/src/main/java/com/example/demo/controller/AnswerController.java

```diff
@@ -0,0 +1,14 @@
+package com.example.demo.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+
+@Controller
+public class AnswerController
+{
+    @GetMapping("/answer")
```

Comment:
Since you are sending the ID of the question in the View answers button, you have to wait for the ID here as well. :)